node/impl/full/eth.go

```diff
@@ -170,8 +169,10 @@ type EthAPI struct {
 	Chain        *store.ChainStore
 	StateManager *stmgr.StateManager
 
+	EthTxHashManager
```

Comment:
```suggestion
	EthTxHashManager EthTxHashManager
```

(no need to export these methods, same with the `MpoolAPI`)